src/main/webapp/js/user-page-loader.js

```diff
@@ -20,105 +20,7 @@ const parameterUsername = urlParams.get('user');
 
 // URL must include ?user=XYZ parameter. If not, redirect to homepage.
 if (!parameterUsername) {
```

Comment:
It doesn't look like we use this if statement, if so should we just remove it?

Reply:
Oops forgot that, thanks for spotting it!